crates/dkg-cli/dkg.bin

```diff
@@ -1 +1 @@
00000000000000000000006000830152602082019050919050565b6000611d92601783612220565b91507f444b472068617320616c726561647920737461727465640000000000000000006000830152602082019050919050565b6000611dd2601e83612220565b91507f6f6e6c79206f776e6572206d617920616c6c6f776c69737420757365727300006000830152602082019050919050565b6000611e12601783612220565b91507f796f7520617265206e6f742072656769737465726564210000000000000000006000830152602082019050919050565b6000611e52600983612220565b91507f444b4720456e64656400000000000000000000000000000000000000000000006000830152602082019050919050565b6000611e92602983612220565b91507f796f75206861766520616c7265616479207075626c697368656420796f75722060008301527f726573706f6e73657300000000000000000000000000000000000000000000006020830152604082019050919050565b6000611ef8601c83612220565b91507f6f6e6c79206f776e6572206d61792073746172742074686520444b47000000006000830152602082019050919050565b611f3481612276565b82525050565b6000602082019050611f4f6000830184611a70565b92915050565b60006020820190508181036000830152611f6f8184611a7f565b905092915050565b60006020820190508181036000830152611f918184611add565b905092915050565b60006020820190508181036000830152611fb38184611b8b565b905092915050565b6000602082019050611fd06000830184611bc4565b92915050565b60006020820190508181036000830152611fef81611bd3565b9050919050565b6000602082019050818103600083015261200f81611c39565b9050919050565b6000602082019050818103600083015261202f81611c79565b9050919050565b6000602082019050818103600083015261204f81611cdf565b9050919050565b6000602082019050818103600083015261206f81611d45565b9050919050565b6000602082019050818103600083015261208f81611d85565b9050919050565b600060208201905081810360008301526120af81611dc5565b9050919050565b600060208201905081810360008301526120cf81611e05565b9050919050565b600060208201905081810360008301526120ef81611e45565b9050919050565b6000602082019050818103600083015261210f81611e85565b9050919050565b6000602082019050818103600083015261212f81611eeb565b9050919050565b600060208201905061214b6000830184611f2b565b92915050565b60006040820190506121666000830185611f2b565b81810360208301526121788184611add565b90509392505050565b6000819050602082019050919050565b6000819050602082019050919050565b600081519050919050565b600081519050919050565b600081519050919050565b6000602082019050919050565b6000602082019050919050565b600082825260208201905092915050565b600082825260208201905092915050565b600082825260208201905092915050565b600082825260208201905092915050565b600082825260208201905092915050565b600061223c82612256565b9050919050565b6000819050612251826122d6565b919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b600061228b82612243565b9050919050565b60005b838110156122b0578082015181840152602081019050612295565b838111156122bf576000848401525b50505050565b6000601f19601f8301169050919050565b600381106122e057fe5b50565b6122ec81612231565b81146122f757600080fd5b50565b61230381612276565b811461230e57600080fd5b5056fea2646970667358221220c0edec4739250c6c309766472d6c9e0278b335effb542994cdc9f5f327a6466e64736f6c63430006060033
```

Comment:
I guess you didn't want to commit this binary file, it's probably a good idea to add it to the gitignore.